test/parallel/test-fs-read-parameters-errors.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const common = require('../common');
+const fixtures = require('../common/fixtures');
+const fs = require('fs');
+const filepath = fixtures.path('x.txt');
+const fd = fs.openSync(filepath, 'r');
+const expected = Buffer.from('xyz\n');
```

Comment:
Is this buffer needed? It seems that only its length is used below.
I would replace it with `const length = 4;`.

Reply:
I have changed it